angr/analyses/reaching_definitions/engine_vex.py

```diff
@@ -492,9 +491,40 @@ def _handle_CCall(self, expr):
     #
 
     def _handle_function(self, func_addr: Optional[DataSet], **kwargs):
+        cc: SimCC = DEFAULT_CC.get(self.arch.name, None)(self.arch)
```

Comment:
You can save an initialization of `SimCC` here by assigning `None` to `cc` first, then initialize it with a default CC if the following `if` block does not identify a good CC for it.

Reply:
Yeah, I changed it on purpose:
  1. It saves up an initialization only when it would be overwritten;
  2. The type hint `cc` is forced to be `Optional[SimCC]` while, after this couple of lines, it can **never** be `None`.

And to me, facilitating the readability (avoiding adding unnecessary `None` checks down the line) outweighed the "maybe one more initialization per function call" save.